crates/redeem/src/lib.rs

```diff
@@ -671,8 +665,8 @@ impl<T: Config> Module<T> {
         Ok(request)
     }
 
-    fn u128_to_dot(x: u128) -> Result<DOT<T>, DispatchError> {
-        TryInto::<DOT<T>>::try_into(x).map_err(|_| Error::<T>::TryIntoIntError.into())
+    fn u128_to_backing(x: u128) -> Result<Backing<T>, DispatchError> {
```

Comment:
question: would it be possible to have a trait that just provides access to these functions via the `Currency` crate so we don't have to re-implement this function in a range of crates?

Reply:
I played around a bit and I don't think it's possible unless we rewrite the trait bounds for `Currency::Balance` which isn't easily done since this is defined upstream.